packages/store/package.json

```diff
@@ -47,7 +47,7 @@
     "dev": "tsup --watch",
     "gas-report": "gas-report --save gas-report.json",
     "lint": "solhint --config ./.solhint.json 'src/**/*.sol'",
-    "test": "vitest typecheck --run && vitest --run --passWithNoTests && forge test",
+    "test": "vitest --run --passWithNoTests && forge test",
```

Comment:
do we intentionally not run typecheck here?

Reply:
Vitest somehow doesn't like the `attest(config).equals(expected)` type, unclear why. Types of all new libs are tested by attest though.